sk0-front/src/app/home/home.page.ts

```diff
@@ -9,7 +10,24 @@ import { IonicModule } from '@ionic/angular';
   standalone: true,
   imports: [IonicModule],
 })
-export class HomePage {
-  userName: string = 'Vendég';
-  constructor() {}
+export class HomePage implements OnInit {
+  userName?: string;
+
+  constructor(
+    private authService: AuthService,
+    private navCtrl: NavController
+  ) {
+    this.userName = this.authService.userName;
+  }
+
+  ngOnInit() {
+    if (!this.authService.isUserLoggedIn) {
+      this.navCtrl.navigateRoot('/login');
+    }
```

Comment:
I would question this part, product-wise. Do we always want to redirect people from the homepage to login?
We are losing the anonymous version of it. (a.k.a. the landing page)

Reply:
I made for now because it is a mobile first "app-like" thingy. but later on when auth guards are here and there, we can refactor it.
Because it will be a kind of time planning app and customer management and later a message handling app, I was wondering what do we want to show in the app without login?